SignalMessaging/environment/OWSSounds.m

```diff
@@ -148,62 +113,12 @@ + (NSString *)displayNameForSound:(OWSSound)sound
             return @"Pulse";
         case OWSSound_Synth:
             return @"Synth";
+        case OWSSound_ClassicNotification:
+            return @"Classic";
```

Comment:
alternative name proposal: "Kickin the Bucket"

Reply:
"Your screen's cracked and your speaker's cracked too."